mbedtls/Cargo.toml

```
@@ -21,7 +21,7 @@ keywords = ["MbedTLS","mbed","TLS","SSL","cryptography"]
 bitflags = "1"
 genio = { version = "0.2.1", default-features = false }
 spin = { version = "0.9.3", default-features = false, optional = true, features = ["mutex","spin_mutex"] }
-serde = { version = "1.0.7", default-features = false }
+serde = { version = "1.0.137", default-features = false }
 serde_derive = "1.0.7"
```

IIUC we can replace the `serde_derive` dep with `features = derive`
```suggestion
serde = { version = "1.0.137", default-features = false, features = ["derive"] }
```